docs/conf.py

```diff
@@ -109,3 +110,6 @@
 
 # Exclude all prompt characters generated by pygments and all console output in copy button
 copybutton_exclude = '.linenos, .gp, .go'
+
+# Convert Central OpenAPI spec to RST
+openapi
```

Comment:
It would be more expected to call a function here. But Python modules and imports are so convoluted I don't want to mess with it right now.